### PR TITLE
style: minimal ui changes

### PR DESCRIPTION
**Pull request by louisedungca**
- remove oauth on sign up form
- change custom color "red" to "warningRed"

### CODE REVIEW

**Review by cyrilcero — APPROVED** [review]
Looks good! Merging!

**Comment by cyrilcero** [comment]
@louisedungca sorry, deployment failed so i think kindly remove the import GoogleOAuth in Register.tsx

File: src/pages/users/auth/Register.tsx
Comment (cyrilcero) [inline src/pages/users/auth/Register.tsx:1]: kindly remove the import GoogleOAuth, i do have a feeling that this causes the build failure

**Review by cyrilcero — CHANGES_REQUESTED** [review]

File: src/pages/users/auth/Register.tsx
Comment (louisedungca) [reply src/pages/users/auth/Register.tsx:1]: This branch was successfully deployed :)

**Review by cyrilcero — APPROVED** [review]
Orayt! Thanks for the changes!

**Pull request merged**